frontend/src/i18n/locales/de/translation.json

```diff
@@ -303,6 +303,7 @@
   "Default Request": "Standardanforderung",
   "Max": "Max",
   "Min": "Min",
+  "Create Namespace": "Namensraum erstellen",
```

Comment:
Namespace in glossary is translated as "Namespace" since it's a technical term. So in here the translation is not correct.
Please leave it empty or update it to using Namespace.

Reply:
change to `"Create Namespace": ""` ?